source/content/guides/support/02-contact-support.md

```diff
@@ -71,12 +69,12 @@ You can download a transcript of your chat with the <Icon icon="arrowDownToLine"
 
 <Alert title="Note" type="info">
 
-Some in-browser ad or tracking blockers can interfere with the **Launch Chat** button. If you don't see it, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has posted more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
+Some in-browser ad or tracking blockers may interfere with the **Start chat** button. If you don't see the button, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has provided more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
 
 </Alert>
 
-## Premium Technical Support and On-Call Services
+## Premium Support Hotline and Emergency Ticket
 
-Diamond and Platinum Account customers can call Pantheon's premium technical support line directly for any technical issues, escalations, site, billing, or overages queries. The phone number can be found in your Workspace, in the Support tab.
+**Diamond** and **Platinum** customers can call the 24/7 Premium Support Hotline for urgent account-related issues. You can find the phone number in the Support tab of your workspace. The phone number can be found in the Support tab of your workspace.
```

Comment:
Keep the previous support limitations. Phone support is not limited to urgent issues, so this could cause confusion. Suggestion
> Diamond and Platinum customers can call Pantheon's 24/7 Premium Support Hotline directly for any technical issues, escalations, site, billing, or overage queries. You can find the phone number in the Support tab of your workspace. Note that this is strictly for filing a ticket, and you will not reach a support engineer by using this method.